File: backend/account_service/internal/dto/task_file_dto.go
package dto

import "github.com/RustamSafiulin/mesh_cloud_computation/backend/account_service/internal/model"

type TaskFileDto struct {
	ID        string `json:"id,omitempty"`
	TaskID    string `json:"task_id,omitempty"`
	Name      string `json:"name,omitempty"`
	Size      int64  `json:"size,omitempty"`
}

func TaskFileDtoFromTaskFile(tf *model.TaskFile) *TaskFileDto {

	return &TaskFileDto{
		ID:     tf.ID.Hex(),
		TaskID: tf.TaskID.Hex(),
		Name:   tf.Name,
		Size:   tf.Size,
	}
}

func TaskFileDtoListFromTaskFileList(taskFiles []model.TaskFile) []TaskFileDto {

	var result []TaskFileDto
	for _, taskFile := range taskFiles {
		taskFileDto := TaskFileDtoFromTaskFile(&taskFile)
		result = append(result, *taskFileDto)
	}

	return result
}

File: desktop_app/MeshDesktopClient/MeshDesktopClientModel/Dto/TaskFileDto.cs
using System;

using Newtonsoft.Json;

namespace MeshDesktopClient.Dto
{
    public class TaskFileDto
    {
        #region Props and Fields

        [JsonProperty(PropertyName = "email")]
        public String Id { get; set; }

        [JsonProperty(PropertyName = "task_id")]
        public String TaskId { get; set; }

        [JsonProperty(PropertyName = "name")]
        public String Name { get; set; }

        [JsonProperty(PropertyName = "size")]
        public Int64 Size;

        #endregion

        public TaskFileDto()
        { }
    }
}